frontend/src/components/ApplicationForm/ApplicationFormPreferences.tsx

```diff
@@ -1,15 +1,22 @@
 import * as React from 'react';
 
 import { Checkbox, Input, Select, SkjemaGruppe } from 'nav-frontend-skjema';
+import { IRoom } from '../ViewRooms';
 
 interface IProps {
   updateApplicationFormData: (item: React.FormEvent) => any;
   partner: boolean;
+  rooms: IRoom[];
 }
 
+const roomsToOptions = (rooms: IRoom[]) => {
+  return rooms.map(room => <option value={room.name} key={room.id}>{room.name}</option>);
```

Comment:
Needs to handle rejection if rooms are empty.

Reply:
rooms is always an array, either empty or with rooms. An empty array will just exit the map and return the empty array. React won't render the empty array, it handles it same as "false"